### PR TITLE
Add `env-vars` subcommand

### LINKED ISSUE #517: CLI commands to manage env vars
There should be some CLI commands to manage arbitrary env vars. Not really sure what they should be since `rtx env` is already used for something else.

`rtx set-env` is one idea I had. Or maybe env-set which will sort better.

### PR DESCRIPTION
**Pull request by smbl64**
This PR adds the `set-env` sub-command. 

It can be used to set or remove environment variables in the local `.rtx.toml` file:
```
rtx env-vars FOO=bar BAZ=qux
rtx env-vars --remove FOO --remove BAZ
```
(`--rm` and `--unset` also work)

User can specify any other TOML file using the `--file` option.

```
rtx env-vars --file .rtx.dev.toml FOO=bar
rtx env-vars --file .rtx.dev.toml --remove FOO
```

Steps left:
- [x] Adding tests for this sub-command 
- [x] Finalizing the command's name and interface (See comments in #517)

Closes #517

### CODE REVIEW

**Comment by smbl64** [comment]
@jdxcode I think this PR is now ready for review :)

File: src/cli/args/env_var.rs [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+        if let Some(arg) = arg {
+            err.insert(
+                ContextKind::InvalidArg,
+                ContextValue::String(arg.to_string()),
+            );
+        }
+        err.insert(
+            ContextKind::InvalidValue,
+            ContextValue::String(value.to_string_lossy().into()),
+        );
```
Comment (jdx) [inline src/cli/args/env_var.rs:48]: nice

File: src/cli/mod.rs [hunk 2]
```diff
@@ -117,6 +119,7 @@ impl Commands {
             Self::Prune(cmd) => cmd.run(config, out),
             Self::Reshim(cmd) => cmd.run(config, out),
             Self::SelfUpdate(cmd) => cmd.run(config, out),
+            Self::EnvVars(cmd) => cmd.run(config, out),
```
Comment (jdx) [inline src/cli/mod.rs]: should be sorted

File: src/cli/env_vars.rs [hunk 3]
```diff
@@ -0,0 +1,120 @@
...
+        }
+
+        for ev in self.env_vars {
+            rtx_toml.update_env(&ev.key, ev.value);
+        }
+        rtx_toml.save()
+    }
+}
+
+fn get_rtx_toml(config: &Config, filename: &str) -> Result<RtxToml> {
```
Comment (jdx) [inline src/cli/env_vars.rs:59]: this should go into a shared function but that can be done later

**Review by jdxcode — APPROVED** [review]
just one minor thing that I can put a fix in for myself

**Comment by jdxcode** [comment]
thanks for this!

**Pull request merged**